lib/charms/kubernetes_charm_libraries/v0/multus.py

```diff
@@ -466,6 +478,19 @@ def _container_security_context_is_set(
         return True
 
 
+class NadConfigChangedEvent(EventBase):
+    """Event triggered when an existing network attachment definition is changed."""
+
+    def __init__(self, handle: Handle):
+        super().__init__(handle)
+
+
+class NadConfigChangedCharmEvents(CharmEvents):
```

Comment:
I'm not sure about the name here. I think we'd be better off naming this class after the lib - something like `KubernetesMultusCharmEvents`.
WDYT?

Reply:
Looks better.
We can use KubernetesMultusCharmEvents and NasConfigChangedEvent as an event source.

Reply:
Done